runtime/Go/antlr/v4/token.go

```diff
@@ -137,7 +137,7 @@ func NewCommonToken(source *TokenSourceCharStreamPair, tokenType, channel, start
 		},
 	}
 
-	if t.source.tokenSource != nil {
+	if t.source != nil && t.source.tokenSource != nil {
```

Comment:
we have call of this function when source is directly `nil`. So I've added this condition